charters/UnityBenchmarkingService/charter.md

```diff
@@ -0,0 +1,64 @@
+
+## Objectives
+
+The Unity Benchmarking Service automate benchmarking for data processing pipelines, enforce standards for consistent reporting for historic comparison and creates opportunity to demonstrate improvements to data processing pipelines using a common benchmarking infrastructure. The Unity Benchmarking Service is used by developers of data processing pipelines for the HCA data processing service and is hosted in a public-facing manner; so anyone can benchmark their data processing pipelines against benchmarked pipelines in the DCP.
+
+## Definitions
+
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project, but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project.  
+__Standardized Pipeline:__ An official pipeline of the HCA project.  
+
+## In-scope
+
+* Benchmarking for standardized HCA data processing pipelines is automated and included in Unity
+* Data used to run pipelines during benchmarking is standardized and enforced to promote comparable analysis runs
+* When multiple pipelines processing the same assay are collected, leaderboards are created to compare performance
+* Documentation for the requirements of pipelines to be benchmarked by the Unity Benchmarking Service
+* Documentation on benchmarking process and outputs
+
+### Scientific "guardrails"
+
+* Pipelines benchmarking standardized HCA data processing pipelines must be reviewed by HCA scientific leadership (Analysis Working Group) and must be treated as standardized data processing pipelines. Changes to these specific benchmarking pipelines will directly affect how standardized HCA data processing pipelines are optimized and comment on what are better scientific results produced by the data processing pipeline they benchmark.
+* Metrics and their use in leaderboards must be reviewed by the HCA scientific leadership (Analysis Working Group).
+
+## Out-of-scope
+
+* Educational training or user assistance on how to create pipelines
+* Recommendations on how to improve pipelines based on benchmarking
+* Pipelines that benchmark Community Pipelines are not required to be included in the Unity benchmarking service (but are allowed). If a Community Pipeline is related to an assay that is neither currently processed by a standardized HCA pipeline nor is being scoped by the Data Processing Pipelines team, then setting up the benchmark will require significant investments which are evaluated based on team priorities. Otherwise, community members may also benchmark Community Pipelines using data and file format standards in the system.
```

Comment:
It is more than this, just because the pipelines confirm to the data and file format standards does not mean that pipeline will be up in Unity, it is per assay and requires having standard data and benchmarking scripts. It is more than running the pipeline. Happy to hear an additional suggestion.